docs/configuration.md

```diff
@@ -0,0 +1,94 @@
+# Configuration
+
+See the default config [here](../config/default.yml)
+
+---
+### allowUnused
+List of modules that are allowed to be unused.
+Passed to `string.match`
+
+Please create an [issue](https://github.com/charlierudolph/dependency-lint/issues)
+anytime you need to use this
+
+Example:
+```yml
+allowUnused:
+  - mocha
+```
+
+---
+### devFilePatterns
+Files used only for development.
+Uses [minimatch](https://github.com/isaacs/minimatch)
+
+Example:
+```yml
+devFilePatterns:
+  - '**/*_test.js'
+```
+
+---
+### devScripts
+List of scripts in your `package.json` used only for development.
+Passed to `string.match`
```

Comment:
Same here. I think if you provide a second example that is a regex this is appropriately covered here. I don't even know of the top of my hat what `string.match` does, I would have to look this up.